node-zerox/src/utils.ts

```diff
@@ -104,3 +108,25 @@ export const convertPdfToImages = async ({
     console.error("Error during PDF conversion:", err);
   }
 };
+
+// Convert each page (from docx) to a png and save that image to tmp
+export const convertDocxToPdf = async ({
+  localPath,
+  tempDir,
+}: {
+  localPath: string;
+  tempDir: string;
+}): Promise<string> => {
+  const inputBuffer = await fs.readFile(localPath);
+  const outputFilename = path.basename(localPath, ".docx") + ".pdf";
+  const outputPath = path.join(tempDir, outputFilename);
+
+  try {
+    const pdfBuffer = await convertAsync(inputBuffer, ".pdf", undefined);
+    await fs.writeFile(outputPath, pdfBuffer);
+    return outputPath;
+  } catch (err) {
+    console.error("Error during DOCX to PDF conversion:", err);
```

Comment:
And then we can update this error message to be `Error converting ${extension} to .pdf:`